src/math/p5.Vector.js

```diff
@@ -1636,6 +1771,32 @@ p5.Vector.add = function add(v1, v2, target) {
   return target;
 };
 
+// Returns a vector remainder when it is divided by another vector
+/**
+ * @method rem
+ * @static
+ * @param  {p5.Vector} v1 dividend <a href="#/p5.Vector">p5.Vector</a>
+ * @param  {p5.Vector} v2 divisor <a href="#/p5.Vector">p5.Vector</a>
+ * @param  {p5.Vector} target the vector to receive the result
```

Comment:
i would suggest removing this third param, and just making a new vector in the function. is there a reason to have the user pass one in?